Library/Dungeon Fantasy/Dungeon Fantasy Adventure 1/Rivals/Mage - Wizard-Scholar.gcs

```diff
@@ -2284,8 +2284,8 @@
 					],
 					"calc": {
 						"level": 13,
-						"parry": "No",
-						"damage": "1d+1 burn/point"
+						"damage": "1d+1 burn/point",
+						"block": "10"
 					}
```

Comment:
A whole lot of your spell attacks have block scores and I don't think they should have

Reply:
I tried refreshing the Wizard-Scholar's spell list from the list in Wizardry Redefined and it's the same from there, so wasn't a change on this character sheet or an outdated entry.

It seems to be that previously not having the block entry meant `"block": "No"` but now it means `"block": "0"`. Parry seems to be the same.